x-pack/plugins/security_solution/public/management/pages/event_filters/view/event_filters_list_page.tsx

```diff
@@ -5,20 +5,58 @@
  * 2.0.
  */
 
-import React, { memo, useCallback } from 'react';
+import React, { memo, useCallback, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Dispatch } from 'redux';
+import { useHistory } from 'react-router-dom';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
+import { AppAction } from '../../../../common/store/actions';
+import { getEventFiltersListPath } from '../../../common/routing';
+import { getCurrentLocation, getActionError, getFormEntry } from '../store/selector';
 import { AdministrationListPage } from '../../../components/administration_list_page';
 import { EventFiltersListEmptyState } from './components/empty';
 import { useEventFiltersNavigateCallback, useEventFiltersSelector } from './hooks';
-import { getCurrentLocation } from '../store/selector';
 import { EventFiltersFlyout } from './components/flyout';
 
 export const EventFiltersListPage = memo(() => {
-  const location = useEventFiltersSelector(getCurrentLocation);
+  const history = useHistory();
+  const dispatch = useDispatch<Dispatch<AppAction>>();
+  const isActionError = useEventFiltersSelector(getActionError);
+  const formEntry = useEventFiltersSelector(getFormEntry);
   const navigateCallback = useEventFiltersNavigateCallback();
+  const location = useEventFiltersSelector(getCurrentLocation);
   const showFlyout = !!location.show;
 
+  // Clean url params if wrong
+  useEffect(() => {
+    if ((location.show === 'edit' && !location.id) || (location.show === 'create' && !!location.id))
```

Comment:
Just thinking from a UX standoint, woudl this be confusing. It is an edge case and one likely not to happen, but just wanted your thoughts on defaulting to "crate" vs a toast message indiciating invalid page or something like that.

Reply:
I tried to reproduce the same behaviour we have in trusted apps. For me, as a user, is better to redirect to a default scenario (in this case to an empty form) instead of showing errors/warnings. But we can ask Bonnie or Caitlin if you want :)

Reply:
I'm ok with that. 👍 Thanks for the response